src/sentry/static/sentry/app/views/settings/project/projectOwnership/ruleBuilder.jsx

```diff
@@ -66,8 +67,8 @@ class RuleBuilder extends React.Component {
     }));
   }
 
-  handleTypeChange = e => {
-    this.setState({type: e[0].value});
+  handleTypeChange = val => {
+    this.setState({type: val});
```

Comment:
very nice. this one seems to work correctly as well

some margin snuck in on the left though
![image](https://user-images.githubusercontent.com/5915546/39021802-507ba894-43e7-11e8-9590-f06060092c7b.png)

Reply:
good catch